tests/data/README.md

```diff
@@ -215,6 +215,96 @@ License: [same as libavif](https://github.com/AOMediaCodec/libavif/blob/main/LIC
 
 Source: Encoded from `paris_icc_exif_xmp.png` using `avifenc -s 10` at commit ed52c1b.
 
+### File [ffffcc-gamma1.6.png](ffffcc-gamma1.6.png)
+
+![](ffffcc-gamma1.6.png)
+
+License: Copyright © 2000-2005 Greg Roelofs.
```

Comment:
Use the license instead of the copyright notice here:

```
Creative Commons Attribution-NonCommercial 2.5 License
```

Note: The copyright symbol © should also be avoided. But it is moot if we put the license instead of the copyright notice here.